### ISSUE TITLE
Scanner fail

### ISSUE THREAD

**Issue by Feredzhanov** [ZeroQI/Absolute-Series-Scanner#199]
Sorry for the issue name. So after updating my folders to [anidb2-xxx] format, I've got the following 2 problems.
The first one is:
![image](https://user-images.githubusercontent.com/27891431/50420290-3dd6e980-083e-11e9-9710-08aa5846c333.png)
what I've noticed is, that if I use the scan function, those are the ones that start to get scanned (It's not only Gundam Seed, Seed Destiny, Magi, Bayoneta etc. as well) and no matter what I try, they never get the metadata.
The other problem is, as far as I understood, using [anidb2-xxx] will split long running shows into seasons, but Naruto Shipuuden still remains as season 1, even if the episodes, that I have are 300-500 and should be in further seasons. So is it some bug, or it's just me understanding the written wrong?

**Comment by ZeroQI** [comment]
" as far as I understood, using [anidb2-xxx] will split long-running shows into seasons" => wrong
- metadata is an agent issue
- anidb2 make AniDB numered series seen as if it was TheTVDB numbered, grouping them if they are mapped to the same tvdb id, and REQUIRES the mapping file to be perfect to do so. 
- tvdb 2 and 3 mode insert seasons in absolutely numbered series. The readme has an example with meitantei conan and one piece mentioned... Do not follow Thetvdb absolute numbering as it insert movies and specials otherwise and mess things up

Please read the readme especially the force id modes.
I cannot understand how you consider anidb2 as a better match than tvdb3 for long-running series...

**Comment by Feredzhanov** [comment]
> I cannot understand how you consider anidb2 as a better match than tvdb3 for long-running series...

Simple, WebAoM sets the folder name, else I'll have to set it manually ;)

Ok, so what can I do for the metadata, or should I post in the HAMA section?

**Comment by ZeroQI** [comment]
Why not use anidb-xxx mode ?
if you need seasons, yes anidb2 sould do that too, but i need full scanner logs (as per readme troubleshooting section) to know why the scanner didn't renumber the eps as it should have

For metadata i need a hama ticket, focused on one series, with the logs as per hama readme troubleshooting section.

**Comment by ZeroQI** [comment]
Working as designed for tvdb4 mode
- series get seasons i guess so [tvdb4-79624] for Naruto Shippuuden is fine
- Logic would dictate to use anidb2 for movies but that would translate in the scanner to tvdb-xxx guid internally, not tvdb4-xxx so not ending up grouped
- "The last: Naruto the movie" is TheTVDB s00e12 so [tvdb4-79624-s00e12] should work
- "Boruto: Naruto the movie" is TheTVDB s00e13 so [tvdb4-79624-s00e13] should work

Never programmed it so people can add seasons and group other items at the same time but had programmed manual offset like above just in case...

**Comment by Feredzhanov** [comment]
Does tvdb4 use ScudLee's mapping list as well, since those aren't mapped there.

**Comment by ZeroQI** [comment]
Tvdb2 and tvdb3 modes takes episodes in sequence and put them in seasons as per TheTVDB. It does keep the absolute episode number in tvdb2 mode while tvdb3 resets every season

Tvdb4 use the all episodes in TheTVDB numbering and use TheTVDB absolute index field to re-order them.

I missed the fact you used tvdb4 as I was thinking about tvdb3 mode. Please replace tvdb4 by tvdb3 in my previous reply.

**Comment by Feredzhanov** [comment]
No, no, what I mean is, if I use tvdb4, does it still read ScudLees's mapping files, or it maps through some kind of other magic (Boruto: Naruto the movie special "The day Naruto became hokage" doesn't match right, even when I've set the folder to [tvdb4-79824-s00e14], and that's a different folder than the main movie itself)

**Comment by Feredzhanov** [comment]
Also, here's other funny thing that I can't make the scanner to make. So besides Boruto: Naruto the movie, I also have the special "The day Naruto became hokage", but no matter how I try i fetches the data for Naruto 2: Bonds. Here's the log.
[Boruto Naruto the Movie [tvdb3-79824-s00e14].scanner.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/2710616/Boruto.Naruto.the.Movie.tvdb3-79824-s
[Boruto Naruto the Movie [tvdb3-79824-s00e14].agent-search.log](https://github.com/ZeroQI/Absolute-Series-Scanner/files/2710617/Boruto.Naruto.the.Movie.tvdb3-79824-s00e14.agent-search.log)
00e14.scanner.log)
![image](https://user-images.githubusercontent.com/27891431/50452938-d0ae7b80-0945-11e9-9adb-8dfb3723fba0.png)

**Comment by ZeroQI** [comment]
Forced ID (series folder) - source: "tvdb3", id: "79824-s00e14"
-------------------------------------------------------------------------------------------------------------------------------------------------------------
Manual file offset - (season: 's00', episode: 'e14') -> (offset_season: '-1', offset_episode: '13')
"Boruto Naruto the Movie [tvdb3-79824]" s00e002 "Word Search" "The Day Naruto Became Hokage" "Boruto Naruto the Movie - S02 - The Day Naruto Became Hokage [Hi10](E16B55C6).mkv"

root cause
- Because of 'S02' it sees it as Special 02 hence s00e02.
- Offset only used for season 1+ so not usable there > My bad, I forgot that one...

it needs the same id to group so try: Boruto Naruto the Movie [tvdb3-79824]/Boruto Naruto the Movie - S00e14 - The Day Naruto Became Hokage [Hi10](E16B55C6).mkv
